File: multiple_inheritance.py
class Vehicle:
    def __init__(self):
        self.model = ""

    def set_brand(self):
        print("Brand: Toyota")


class Car:
    def __init__(self, price):
        self.price = price


class Cycle:
    def __init__(self, model, price):
        self.vehicle = Vehicle()
        self.car = Car(price)
        self.model = model

    def set_brand(self):
        self.vehicle.set_brand()


obj = Cycle("Cycle", 1000)
obj.set_brand()

File: multiple_inheritance.cpp
#include<iostream>
using namespace std;

class Vehicle {
public:
    string model;

    void setBrand() {
        cout << "Brand : " << "Toyota" << endl;
    }
};

class car {
public:
    int price;
    car(int price) {
        this->price = price;
    }
};

class Cycle : public Vehicle, public car {
public:
    Cycle(string model, int price) : Vehicle(), car(price) {
        this->model = model;
    }
};

int main() {
    Cycle obj("Cycle", 1000);  
    obj.setBrand(); 
}
